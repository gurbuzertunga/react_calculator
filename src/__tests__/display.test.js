import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Display from '../components/display';

describe('checks if Display component renders correctly', () => {
  test('should render correctly', () => {
    const tree = renderer.create(<Display />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('should renders correct value when edited', () => {
    const result = '23';

    render(<Display result={result} />);
    const text = screen.getByText('23');
    expect(text).toBeInTheDocument();
  });
});
