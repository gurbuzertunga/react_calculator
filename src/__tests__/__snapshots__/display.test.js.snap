// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checks if Display component renders correctly should render correctly 1`] = `
<div
  className="display"
>
  div
</div>
`;
